docs/resources/database_postgresql.md

```diff
@@ -0,0 +1,76 @@
+# Resource: <resource name> delphix_database_postgresql
+
+In Delphix, a databsae/source is required in the environment using which a dsource can be created.
+
+## System Requirements
+
+* Data Control Tower v14.0.1+ is required for dSource management. Lower versions are not supported.
+* This Databse PostgresSql Resource only supports POSTGRES , This resource does not support Oracle, SQL Server, or SAP ASE.
```

Comment:
"The Database PostgreSQL resource only supports the Delphix PostgreSQL database type (POSTGRES). This resource does not support Oracle, SQL Server, or SAP ASE."